packages/react-interactions/events/src/dom/__tests__/Input-test.internal.js

```diff
@@ -140,8 +140,15 @@ describe('Input event responder', () => {
       ref.current.dispatchEvent(
         new Event('input', {bubbles: true, cancelable: true}),
       );
-      expect(onChangeCalled).toBe(0);
-      expect(onValueChangeCalled).toBe(0);
+
+      if (ReactFeatureFlags.disableInputAttributeSyncing) {
+        // TODO: figure out why. This might be a bug.
```

Comment:
This isn't new. Copypaste from ChangeEventPlugin test from which this was copied.